src/tests/topp/MRMTransitionGroupPicker_1_output.featureXML

```diff
@@ -194,6 +205,7 @@
 					<UserParam type="string" name="native_id" value="60352_NSFVSYENFYQNHENAEIR/3_y10"/>
 					<UserParam type="float" name="peak_apex_int" value="403"/>
 					<UserParam type="float" name="total_xic" value="4746"/>
+					<UserParam type="float" name="width_at_50" value="17.0700000000002"/>
```

Comment:
ok, this is the "real" peak, and it makes a ton of sense, see 
![screenshot from 2018-09-28 12-17-25](https://user-images.githubusercontent.com/464014/46220609-a8988780-c318-11e8-95cc-50f634a9480d.png)